.github/workflows/build-accumulo.yml

```
@@ -0,0 +1,107 @@
+      accumuloRepo:
+        required: true
+        default: "apache/accumulo"
+        description: "Accumulo Repo to use. Expected to be at Github. Example: apache/accumulo"
+      deployAccumulo:
+        required: true
+        default: "false"
+        description: "Set to false if this accumulo version has already been pushed to Github Packages"
+
+# Defines two custom environment variables for the workflow. These are used for the Container registry domain, and a name for the Docker image that this workflow builds.
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository.lowercase }}
+  ACCUMULO_JAVA_VERSION: '17'
+  DATAWAVE_JAVA_VERSION: '11'
+  JAVA_DISTRIBUTION: 'zulu' #This is the default on v1 of the action for 1.8
+  USER_NAME: ${{ secrets.GHCR_WRITE_USER_NAME }}
+  ACCESS_TOKEN: ${{ secrets.GHCR_WRITE_ACCESS_TOKEN }}
+
+jobs:
+  build-and-deploy-accumulo:
+    runs-on: ubuntu-latest
+    # Sets the permissions granted to the `GITHUB_TOKEN` for the actions in this job.
+    permissions:
+      contents: read
+      packages: write
+      #
+    steps:
+    - name: Checkout DataWave
+      uses: actions/checkout@v4
+      with:
+        path: datawave
+
+    - name: Checkout Accumulo
+      uses: actions/checkout@v4
+      id: accumuloCheckout
+      with:
+        repository: ${{ github.event.inputs.accumuloRepo }}
+        path: accumulo
+        ref: ${{ github.event.inputs.accumuloBranch }}
+
+    - name: Set up JDK ${{env.ACCUMULO_JAVA_VERSION}}
+      uses: actions/setup-java@v4
+      with:
+        distribution: ${{env.JAVA_DISTRIBUTION}}
+        java-version: ${{env.ACCUMULO_JAVA_VERSION}}
+        cache: 'maven'
+    - uses: actions/cache@v4
+      with:
+        path: ~/.m2/repository
+        key: ${{ runner.os }}-maven-build-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          ${{ runner.os }}-maven-build-
+          ${{ runner.os }}-maven-format-
+          ${{ runner.os }}-maven-
+    - run: echo "ACCUMULO_JAVA=$JAVA_HOME" >> $GITHUB_ENV
+    - name: Set up JDK ${{env.DATAWAVE_JAVA_VERSION}}
+      uses: actions/setup-java@v4
+      with:
+        distribution: ${{env.JAVA_DISTRIBUTION}}
+        java-version: ${{env.DATAWAVE_JAVA_VERSION}}
+        cache: 'maven'
+    - run: echo "DATAWAVE_JAVA=$JAVA_HOME" >> $GITHUB_ENV
+
+    - name: Get Accumulo Version
+      id: get-accumulo-version
+      run: |
+        export JAVA_HOME=$ACCUMULO_JAVA
+        cd $GITHUB_WORKSPACE/accumulo
+        mvn build-helper:parse-version versions:set -DgenerateBackupPoms=false -DnewVersion=\${parsedVersion.majorVersion}.\${parsedVersion.minorVersion}.\${parsedVersion.incrementalVersion}-dwv-$(git rev-parse --short HEAD)
+        export newVersion=$(mvn -q help:evaluate -DforceStdout -Dexpression=project.version)
+        echo accumuloVersion=$newVersion >> $GITHUB_OUTPUT
+    - name: Deploy Accumulo
+      if: ${{ ( github.event.inputs.deployAccumulo || 'false' )  == 'true'}}
```

```suggestion
      if: ${{ github.event.inputs.deployAccumulo }}
```

I think this `||` piece is already handled by the default value with the recent changes to make it a required property with a default.